include/swift/Serialization/ModuleFormat.h

```diff
@@ -52,7 +52,7 @@ const uint16_t SWIFTMODULE_VERSION_MAJOR = 0;
 /// describe what change you made. The content of this comment isn't important;
 /// it just ensures a conflict if two people change the module format.
 /// Don't worry about adhering to the 80-column limit for this line.
-const uint16_t SWIFTMODULE_VERSION_MINOR = 456; // Last change: encode depth in generic param XREFs
+const uint16_t SWIFTMODULE_VERSION_MINOR = 458; // Last change: enrich FILE_DEPENDENCY records.
```

Comment:
We skipped 457?

Reply:
It's at an intermediate state in the patch series. When introducing dep names but not yet times-and-sizes. I figured it's bad form to have a change to format that doesn't bump number in lock step even if they're both in the same PR.

Reply:
Ah, seems reasonable. (I didn't review commit-by-commit this time.) Thanks for the explanation!